### ISSUE TITLE
collapse random table of unused info at bottom of data record 

### ISSUE THREAD

**Issue by takurkowski** [ua-snap/ckanext-snap_theme#4]

**Comment by brucecrevensten** [comment]
For the record: that "random table of unused info" is our metadata.  :cake:

Agreed that it's a bit rough.  I've got it collapsing/expanding now, but it's a pretty dumb implementation, may want to look at the approach that Data.gov took: show part of the table by default (which has some pretty good "first class citizens" in the metadata, with an option to expand more.

**Comment by takurkowski** [comment]
yes, a more elegant solution would allow us to display our license type assigned in GeoN in a nice way.

**Comment by takurkowski** [comment]
yes, I like what data.gov has done, let's copy that!
http://catalog.data.gov/dataset/federal-logistics-information-system-web-search-webflis

some of this is listed as a text block when you expand the table, but if we eliminate that work, and instead display the following attributes in a small table first:

...tag list here...

...small table with these attributes...
About this Dataset
license
license_url
spatial-reference-system
spatial-resolution
spatial-resolution-units
temporal-extent-begin
temporal-extent-end
credits
contact-email

...show more...
